### DISCUSSION TITLE
CopyQ with m1 Support crashes

### DISCUSSION THREAD

**Discussion by tom9123** [hluk/CopyQ#2772] [Q&A] [answered]
The application with version 10 as well as version 9 does not run on my MacOS. The program crashes after starting

any Ideas why?

![m1maxVersion](https://github.com/hluk/CopyQ/assets/135061009/ed4f65ff-2833-4889-a97b-e38ada41eeec)
![Bildschirmfoto 2024-07-11 um 13 49 41](https://github.com/hluk/CopyQ/assets/135061009/feba62af-bbbc-4fe7-b085-7f96b787cf28)

**Comment by hluk** [comment, marked as answer]
See: https://github.com/hluk/CopyQ/issues/2710

And solution: https://github.com/hluk/CopyQ/issues/2710#issuecomment-2128391676

**Reply by tom9123** [reply]
Thx for help
